org.ekstep.collaborator-1.2/editor/collaboratorApp.js

```diff
@@ -322,10 +322,13 @@ angular.module('collaboratorApp', ['angular-inview'])
                     $scope.searchRes.errorMessage = "Oops! Something went wrong. Please try again later.";
                 } else {
                     $scope.searchRes.searchStatus = "end";
-
+                    
                     /* istanbul ignore else */
                     if (res.data.result.response.count) {
-
+                        
+                        _.forEach(res.data.result.response.content, function( object, index) {
+                            (object.lastName == null) ? res.data.result.response.content[index].lastName = "" : "" ;
```

Comment:
res.data.result.response.content[index].lastName = "" :""
this is not correct. You dont need ternary operation for this.

Reply:
Removed  ternary operator using if block instead. @vinukumar-vs